### PR TITLE
Honda: HR-V 2023 radarless support

### PR DESCRIPTION
**Pull request by Ktrtkd**
**Description** Adds Brazilian HRV/Vezel 2023Model
**Verification** Fingerprint
**Route**
Test Route: 147613502316e718|2023-02-14--19-59-09

<!--- ***** Template: Car port *****

**Checklist**
- [ ] added entry to CarInfo in selfdrive/car/*/values.py and ran `selfdrive/car/docs.py` to generate new docs
- [ ] test route added to [routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py)
- [ ] route with openpilot:
- [ ] route with stock system:

### CODE REVIEW

File: selfdrive/car/honda/values.py [hunk 1]
```diff
@@ -1517,6 +1545,7 @@ def init_make(self, CP: car.CarParams):
   CAR.INSIGHT: dbc_dict('honda_insight_ex_2019_can_generated', None),
   CAR.HONDA_E: dbc_dict('acura_rdx_2020_can_generated', None),
   CAR.CIVIC_2022: dbc_dict('honda_civic_ex_2022_can_generated', None),
+  CAR.HRV_2023: dbc_dict('honda_hrv_2023_can_generated', None),
```
Comment (sshane) [inline selfdrive/car/honda/values.py:1548]: Where is this DBC? Can you use the radarless Civic one?

**Comment by sshane** [comment]
Thanks for the PR! Do you know if the radarless aspect is different in the US? If not, we can remove that part in the car docs

File: selfdrive/car/honda/values.py (hunk 1 above)
Comment (Ktrtkd) [reply selfdrive/car/honda/values.py:1548]: Probably still not sure. This HR-V is based on Honda Vezel. Not the US model.  Japan/Thailand/Brazil. This data comes from a Brazillian HR-V

**Comment by Ktrtkd** [comment]
> Thanks for the PR! Do you know if the radarless aspect is different in the US? If not, we can remove that part in the car docs

I think its similar, I might made a mistake to pull this here. Still need to test properly in my local branch.

File: selfdrive/car/honda/values.py (hunk 1 above)
Comment (Ktrtkd) [reply selfdrive/car/honda/values.py]: I will try the civic2022 dbc.

**Comment by sshane** [comment]
Any reason you closed the PR?

**Comment by Ktrtkd** [comment]
> 

Yup. Re-doing everthing from scratch by adding under Civic 22 in another repo and doing proper tests. 
But having the same results. Not sure how to handle this, commented on https://github.com/commaai/openpilot/pull/27345

**Pull request closed without merging**